shared/metrics/src/main/java/io/pravega/shared/metrics/DynamicLoggerImpl.java

```diff
@@ -103,6 +103,7 @@ public Counter call() throws Exception {
                 }
             });
             counter.add(delta);
+            log.trace("Metrics: increased counter {} by {} ", keys.getCacheKey(), delta);
```

Comment:
Thanks for changing to trace level, but that's not exactly what I had in mind. To quote myself:

> If you change to trace, then consider using `LoggerHelpers.traceEnter` and `LoggerHelper.traceLeave`.


Main reference:
https://github.com/pravega/pravega/blob/master/common/src/main/java/io/pravega/common/LoggerHelpers.java#L19

and examples:
https://github.com/pravega/pravega/search?q=LoggerHelpers&unscoped_q=LoggerHelpers

Reply:
Fixed